subsys/power/device.c

```diff
@@ -45,6 +45,16 @@ static const char *const core_devices[] = {
 static const char *const core_devices[] = {
 	"",
 };
+#elif defined(CONFIG_SOC_SERIES_STM32L4X)
+#define MAX_PM_DEVICES	15
+#define NUM_CORE_DEVICES	4
+#define MAX_DEV_NAME_LEN	16
+static const char core_devices[NUM_CORE_DEVICES][MAX_DEV_NAME_LEN] = {
+	"CLOCK_32K",
+	"CLOCK_16M",
+	"sys_clock",
+	"UART_0",
```

Comment:
These are not STM32 devices, should this list left empty for now?

Reply:
changed to 
#define MAX_PM_DEVICES	1
static const char *const core_devices[] = {
	"",

Reply:
`"sys_clock"` was actually correct, it corresponds to file `drivers/timer/sys_clock_init.c` which maps to the LPTIM in that case.

Reply:
changing to :
```
#elif defined(CONFIG_SOC_SERIES_STM32L4X) || defined(CONFIG_SOC_SERIES_STM32WBX)
#define MAX_PM_DEVICES	2
static const char *const core_devices[] = {
	"sys_clock",
};
```